pyopensn/src/aquad.cpp

```diff
@@ -0,0 +1,132 @@
+/*
+ * Created on Sun, February 23
+ *
+ * Copyright (c) 2025 The OpenSn Authors <https://open-sn.github.io/opensn/>
+ */
+
+#include "pyapi.hpp"
+
+#include <memory>     // std::shared_ptr, std::make_shared
```

Comment:
Do not do the comment which explains why you `#include` a header file. We don't do that anywhere, so keep the code consistent with the rest of the code base.